src/transformers/pipelines/image_segmentation.py

```diff
@@ -32,7 +32,7 @@ class ImageSegmentationPipeline(Pipeline):
     Image segmentation pipeline using any `AutoModelForXXXSegmentation`. This pipeline predicts masks of objects and
     their classes.
 
-    This image segmentation pipeline can currently be loaded from [`pipeline`] using the following task identifier:
+    This image segmentation pipeline can currently be loaded from [`pipeline`] using the following subtask identifier:
```

Comment:
I think `image-segmentation` is still the `task` identifier. 
```suggestion
    This image segmentation pipeline can currently be loaded from [`pipeline`] using the following task identifier:
```

Reply:
Too agressive replace. Thanks for the catch !

Reply:
Done.